### PR TITLE
Custom fields demos

### PR DESCRIPTION
**Pull request by MadeByMike**
Adds a new demo to help people creating custom fields.

Add an updated star field and a multi-check field.

image

### CODE REVIEW

File: demo-projects/custom-fields/fields/MultiCheck/views/Controller.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const FieldController = require('@keystonejs/fields/Controller').default;
```
Comment (mitchellhamilton) [inline demo-projects/custom-fields/fields/MultiCheck/views/Controller.js:1]: Why are you using CommonJS here?

Comment (MadeByMike) [reply demo-projects/custom-fields/fields/MultiCheck/views/Controller.js:1]: Because it borked out when I didn't 😆

Comment (mitchellhamilton) [reply demo-projects/custom-fields/fields/MultiCheck/views/Controller.js:1]: That's very strange. The Controller should be able to be ESM

**Comment by pahaz** [comment]
https://github.com/keystonejs/keystone/issues/2851 related to this PR

**Pull request merged**